src/http/mod.rs

```diff
@@ -0,0 +1,255 @@
+use self::{
+    usage::Usage,
+    user_input::{Scheme, UserInput},
+};
+use crate::results::WorkerResult;
+use anyhow::anyhow;
+use futures_util::{stream::FuturesUnordered, TryFutureExt};
+use http::{
+    header::{self, HeaderMap},
+    Request,
+};
+use hyper::{
+    client::conn::{self, SendRequest},
+    Body,
+};
+use std::{collections::HashMap, net::SocketAddr, time::Duration};
+use tokio::{
+    io::{AsyncRead, AsyncWrite},
+    net::TcpStream,
+    task::JoinHandle,
+    time::{error::Elapsed, sleep, timeout_at, Instant},
+};
+use tower::{util::ServiceExt, Service};
```

Comment:
I think rustfmt needs to be ran on these changes. 

My general style with imports is ordered and separated by whitespace:

- `std`
- `3rd party deps` 
- `crate/super/self` imports